### DISCUSSION TITLE
Firmware to big for device.

### DISCUSSION THREAD

**Discussion by mattlward** [benzino77/tasmocompiler#387] [General]
Not sure if I am missing something here or not, but the new firmware compiles fine but the file is to big and fails to install.  

From current Tasmota:
<html><body>

ESP Chip Id | 8801075 (ESP8285N08)
-- | --
0x144051 (DOUT)
1024 KB
1024 KB
638 KB
364 KB
24.8 KB

</body>
</html>

Device is a Topgreener energy monitoring plug.  I have tried generic and Wemos and I am using the .gz file upload option.

**Comment by benzino77** [comment]
Hi,

I'm not sure this is the best place to ask that question. You should rather ask that question on Tasmota Discord [channel](https://discord.gg/zc6az8J) or directly on Tasmota  discussion [page](https://github.com/arendst/Tasmota/discussions).

What I can advise is to flash minimal firmaware first before you upload bin file produced by TasmoCompiler. The board you are using is 1MB so there is not so much place to do OTA without intermediate "minimal" firmware flushing.
